cinema/views.py

```diff
@@ -35,7 +34,6 @@ class GenreViewSet(
 ):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
-    authentication_classes = (TokenAuthentication,)
     permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
```

Comment:
you can set default permission in REST_FRAMEWORK like DEFAULT_PERMISSION_CLASSES and remove code duplication